DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/RevitConverterToSpeckle.cs

```diff
@@ -45,7 +45,17 @@ public Base Convert(object target)
       // POC: is this the right place?
       result.applicationId = element.UniqueId;
 
-      _convertedObjectsCache.AddConvertedBase(element.UniqueId, result);
+      try
+      {
+        _convertedObjectsCache.AddConvertedBase(element.UniqueId, result);
+      }
+      catch (ArgumentException)
+      {
+        // POC: object converted multiple times
+        // we are doing this all the time in our current converter, and the serializer is fixing it for us.
+        // so for now, I am just silencing this exception
+        // https://spockle.atlassian.net/browse/CNX-9402
+      }
```

Comment:
Why aren't we just coding the cache to not explode?

Something like `HasConvertedBase()` or refactor `AddConvertedBase()` to return true if it succeeds to add the object to the cache?

We're also using `Add` in the cache super naively, assuming an object will never be set beforehand, but this PR's existence means that this assumption is not correct.

I would argue this is not the right fix, and instead the cache should be made less easy to explode (it is a cache after all, which should be a utility thing that doesn't really need to exist to perform the main logic correctly)